server/utils/email/digest.ts

```diff
@@ -0,0 +1,81 @@
+import pick from 'lodash.pick';
+import omit from 'lodash.omit';
+import { GroupedActivityItems } from 'client/components/Email';
+
+import { ActivityItem } from 'types';
+import { fetchActivityItems } from 'server/activityItem/fetch';
+
+const getAffectedObject = (item) =>
+	item.pubId
+		? { id: item.pubId, title: item.payload.pub.title }
```

Comment:
⚠️ These payload titles are sometimes going to be out of date...this is frequently true with Pubs, which start out as "Untitled Pub..." but are usually quickly renamed. So we need to look inside the `associations` from `fetchActivityItems()` to get the latest title:

```ts
const getAffectedObject = (item: ActivityItem, associations: ActivityAssociations) => {
	if (item.pubId) {
		const pub = associations.pub[item.pubId];
		return { id: item.pubId, title: pub.title };
	}
	// ...and so on
};
```